src/components/audio-zone.js

```diff
@@ -0,0 +1,106 @@
+const DEBUG_BBAA_COLOR = 0x49ef4;
+
+const debugMaterial = new THREE.MeshBasicMaterial({
+  color: DEBUG_BBAA_COLOR,
+  transparent: true,
+  opacity: 0.25,
+  side: THREE.DoubleSide
+});
+
+/**
+ * Represents an 3D box area in the audio-zones-system that can contain audio-zone-entities.
+ * I has an audio-params components which values are used to override the audio sources audio properties 
+ * based on the source's and listener's position. It can be of inOut or/and outIn types.
+    inOut: applies this zone's audio-params to an audio-zone-source when the source is inside and listener is outside.
+    i.e. You want to mute audio sources inside the audio zone when the listener is outside.
+    outIn: applies this zone's audio-params to the an audio-zone-source when the listener is inside and the source is outside.
+    i.e. You want to mute audio sources from outside the audio zone when the listener is inside.
+ */
+AFRAME.registerComponent("audio-zone", {
+  schema: {
+    enabled: { default: true },
+    inOut: { default: true },
+    outIn: { default: true },
+    debuggable: { default: true }
+  },
+
+  init() {
+    this.currentPos = new THREE.Vector3();
+    this.lastPosition = new THREE.Vector3();
+    this.worldBoundingBox = new THREE.Box3();
+    this.object3D = this.el.object3D;
+
+    const bbox = new THREE.Box3();
+    bbox.setFromObject(this.object3D);
+    const size = new THREE.Vector3();
+    bbox.getSize(size);
+    // Adjust the box scale to the bounding box size
+    if (size.length() > 0) {
+      this.object3D.scale.set(size.x, size.y, size.z);
+    }
+
+    for (let i = this.object3D.children.length - 1; i >= 0; i--) {
+      this.object3D.children[i].visible = false;
```

Comment:
Instead of defining a size manually in the blender exporter and Spoke as we do with the trigger volume, I'm just setting the zones' bounding box to the object bounding box. When exporting the object from blender and dropping in the scene the object might have a mesh int he hierarchy so I'm hiding the possible children there.

I'm happy to discuss this further, I'm pretty sure that this can be done in a better way.